requirements-min.txt

```diff
@@ -1,5 +1,5 @@
 # Requirements for minimal tests
 -r requirements.txt
-setuptools>=50.3.0
+setuptools>=50.3.0,!=60.0.0
```

Comment:
Hi @wyli ,

Thanks for your quick fix.
Could you please help describe the root cause?

Thanks in advance.

Reply:
I didn't look into the details but there are some discussions in the issue list https://github.com/pypa/setuptools/issues

Reply:
Could you please help add some comments here? I am afraid we don't remember why add it in the future...
Others look good to me.

Thanks.

Reply:
I think we can track it with git blame and all the github ticket and pull request information. https://github.com/Project-MONAI/MONAI/blame/372f149b3472d8ef1d0a5d022c8ab34bc754b035/requirements-min.txt